### DISCUSSION TITLE
[Suggestion]Separates wither and normal skeletons into different entities

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#460] [Improvements]
I remember the modern version is like this

**Comment by Roadhog360** [comment]
Modern versions are like that yes, but in 1.7.10 there is no point and it'd only create issues when mods expect it to be a version of the normal skeleton.

If you want this because of spawn eggs that is possible.

**Reply by Omgise** [reply]
It is because the spawn wants this

**Reply by GregoriusT** [reply]
what is "the spawn" in this context? you mean the mob spawner?

**Reply by Omgise** [reply]
Yes

**Reply by Roadhog360** [reply]
Mob spawners can accept NBT for mobs; if I add a Wither Skeleton spawn egg I can make it do that.
